### PR TITLE
start branch

### PR DESCRIPTION
**Pull request by satishighs**
This is drumstick project ................

This is open source project, let me know if you would like to contribute ........

### CODE REVIEW

**Comment by vatsalgarg** [comment]
Good GOiing!!!!

File: firstRow/satish-drumstickfarm.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function drumStickSapling() {
+	console.log("Grow trees");
+	console.log("Sell drumsticks");
+	console.log("Make profits");
+	console.log("Take vacation");
```
Comment (githubteacher) [inline firstRow/satish-drumstickfarm.js]: Looks good! I also need a vacation

**Comment by githubteacher** [comment]
@satishighs great change! looks good to me. Please merge or add something for more practice.

**Comment by satishighs** [comment]
definitly

**Pull request merged**